bulk/eng_bulk_load-AssetRecord.csv

```diff
@@ -1,544 +1,545 @@
-ASSET_UID,LEGACY_ASSET_UID,TYPE,Mobile,DESCRIPTION OF EQUIPMENT,MIO_Inventory_Description,Manufacturer,Model,Manufacturer's Serial No./Other Identifier,Firmware Version,ACQUISITION DATE,ORIGINAL COST,comments,MIO
-CGCON-AUVPNR-06263,,Sensor,1,Mobile Asset Controller for Coastal AUV,A6263 Controller,Hydroid,AUV,6263,,,,,Woods Hole Oceanographic Institution
-CGCON-AUVPNR-06264,,Sensor,1,Mobile Asset Controller for Coastal AUV,A6264 Controller,Hydroid,AUV,6264,,,,,Woods Hole Oceanographic Institution
-CGCON-BCPM01-00001,OL000483,Sensor,0,Surface Buoy Communications and Power Manager,Communications and Power Manager,WHOI,CPM,0001,,,,CP01CNSM-00001-CPM1,Woods Hole Oceanographic Institution
```

Comment:
Your update removed all of the leading zeros from our S/Ns.  Did you mean to do that?  Or did you do this in Excel and it just decided that leading zeros are not important?

Reply:
OH wow good catch.  The whole csv is highlighted for me instead of just specific changes.  Didn't catch that.  And THAT'S why Collin wanted you to check these too!

Reply:
Sorry that was Excel being stupid with it's assumptions.